.github/ISSUE_TEMPLATE/offboarding-request.md

```diff
@@ -44,27 +44,33 @@ Please provide the following information about the individual being offboarded:
 
 _The following steps are performed by the **Platform Support** team. Detailed instructions found [here](https://vfs.atlassian.net/wiki/spaces/OT/pages/2097545323/Offboard+Team+Member)_
 
- - [ ] Requested removal from DSVA Slack (if applicable. Search for them in Slack)
+### Support Tier 1 🌺
```

Comment:
What if right below this line we had [Instructions]() (but we actually link to the instructions 😃)?

Reply:
Created a line below to provide a link to instructions for Tier 1